jbmc/src/java_bytecode/library/converter.cpp

```diff
@@ -66,7 +66,7 @@ int main(int argc, char *argv[])
     printf("\n");
   }
 
-  std::cout << "\n#define " << varname <<  "_SIZE " << size << "\n\n";
+  std::cout << "\n#define " << varname <<  "_SIZE " << size << "\n";
```

Comment:
I know you mentioned this change in the PR description, and I don't have strong feels whether it belongs in the PR or not, but if it's in this PR it should probably be in a commit of it's own.